gap/attributes/attr.gi

```diff
@@ -407,6 +407,9 @@ end);
 InstallMethod(StructureDescription, "for a group as semigroup",
 [IsGroupAsSemigroup],
 function(S)
+  if IsGroup(S) then
+    TryNextMethod();
+  fi;
   return StructureDescription(Range(IsomorphismPermGroup(S)));
 end);
 
```

Comment:
Shouldn't this whole chunk be in a separate commit/PR since it doesn't have anything to do with this change? Or does it?